### PR TITLE
#244: Drops JQuery from Test Helper

### PR DESCRIPTION
**Pull request by bukhtiyarov-a-v**
@wbotelhos please take a look. This removes JQuery from dependencies.

### CODE REVIEW

File: src/raty.js [hunk 1]
```diff
@@ -707,6 +707,7 @@ class Raty {
     } else {
       target.textContent = score;
     }
+    console.log(target.value);
```
Comment (wbotelhos) [inline src/raty.js:710]: 🔥

Comment (bukhtiyarov-a-v) [reply src/raty.js]: I removed this line, it was left after debugging

File: yarn.lock [hunk 2]
```diff
@@ -4169,11 +4169,6 @@ jest@^29.1.2:
     import-local "^3.0.2"
     jest-cli "^29.1.2"
 
-jquery@^3.6.1:
```
Comment (Fyzu) [inline yarn.lock:4172]: 🎉

**Comment by wbotelhos** [comment]
Thanks, @bukhtiyarov-a-v ! ❤️

**Pull request merged**